com.kavilgroup.gestimator.json

```diff
@@ -0,0 +1,106 @@
+{
+    "app-id": "com.kavilgroup.gestimator",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "command": "gestimator.py",
+    "finish-args": [
+           "--socket=x11",
+           "--socket=wayland",
+           "--socket=fallback-x11",
+           "--share=network",
+           "--filesystem=host",
```

Comment:
What the apps needs the whole host permission for?

Reply:
Program needs to export a spread sheet as output.

Reply:
You can open and save files without any holes on the filesystem by using portals.

Reply:
If you use GtkFileChooserNative rather than GtkFileChooser then it should Just Work without any file holes in the sandbox. https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html

Reply:
Thanks for the advice. I have made corrections for the program to use portals. The additional permissions can now be removed.